src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FtpaRespondentPreparer.java

```diff
@@ -82,13 +73,15 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
             asylumCase.clear(FTPA_RESPONDENT_EVIDENCE_DOCUMENTS);
         }
 
-        final Optional<String> mayBeAppealDate = asylumCase.read(APPEAL_DATE);
-
-        Optional<OutOfCountryDecisionType> maybeOutOfCountryDecisionType = asylumCase.read(OUT_OF_COUNTRY_DECISION_TYPE, OutOfCountryDecisionType.class);
+        Optional<String> optionalFtpaApplicationDeadline = asylumCase.read(FTPA_APPLICATION_DEADLINE);
+        if (optionalFtpaApplicationDeadline.isEmpty()) {
```

Comment:
Existing functionality where appealDate missing defaults ftpaOutOfTime to No has been changed to throw a RequiredFieldMissingException (when ftpaApplicationDeadline is missing instead of appealDate as we use ftpaApplicationDeadline to determine ftpaOutOfTime now as per this PR ).